### PR TITLE
[5.1] Router group helper

### PR DESCRIPTION
**Pull request by fredriklaason**

### CODE REVIEW

File: src/Illuminate/Foundation/helpers.php [hunk 1]
```diff
@@ -292,6 +292,20 @@ function get($uri, $action)
     }
 }
 
+if (!function_exists('group')) {
+    /**
+     * Create a route group with shared attributes.
+     *
+     * @param  array     $attributes
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/helpers.php]: Don't align this please.

**Comment by taylorotwell** [comment]
I've thought about this, but the name `group` is just too ambigious and common for a helper I think.

**Comment by lukasgeiter** [comment]
@taylorotwell So are `get()` and `post()`... :wink:

**Comment by fredriklaason** [comment]
@lukasgeiter @taylorotwell Indeed

**Comment by GrahamCampbell** [comment]
I don't think GET and POST are anywhere near as ambiguous as group.

**Comment by GrahamCampbell** [comment]
If you really want this function, it's pretty easy to register it yourself in your app. :)

**Comment by lukasgeiter** [comment]
If you write it all in capitals or if it appears in a routing context. However the function name itself is just a very commonly used english verb (and noun as well in the case of `post`). I'm not for or against adding a `group()` helper, I just wanted to point out that in my opinion some helpers are already very ambiguous.

**Pull request closed without merging**